UITests/BookmarksPanelTests.swift

```diff
@@ -62,4 +69,43 @@ class BookmarksPanelTests: KIFTestCase {
         tester().waitForView(withAccessibilityLabel: "https://people-mozilla.org/~npark")
         
     }
+
+    func testRootHasLocalAndBuffer() {
+        let application = UIApplication.shared
+        guard let delegate = application.delegate as? TestAppDelegate else {
+            XCTFail("Couldn't get app delegate.")
+            return
+        }
+
+        let profile = delegate.getProfile(application)
+        guard let bookmarks = profile.bookmarks as? BookmarkBufferStorage else {
```

Comment:
Can we share this logic with `createSomeBufferBookmarks`, perhaps by breaking out a `getApplicationBookmarks` method and giving an optional argument to the helper?